xc/common/cmake/arch_define.cmake

```diff
@@ -90,6 +90,8 @@ function(ADD_XC_ARCH_DEFINE)
       add_file_target(FILE ${VPR_CELLS_SIM} ABSOLUTE)
   endif ()
 
+  get_target_property_required(XCFASM env XCFASM)
```

Comment:
Why use the xc-fasm submodule versus the package?  I'd argue that we should use the xc-fasm package, but the prjxray-db submodule, but *shurg*?

Reply:
What you mean by xc-fasm package? Currently it is installed via pip in the requirements.txt, I have followed the same ways as the other python packages we are using (e.g. fasm, python-sdf-timing).

Reply:
Sorry, I agree.  I was thinking xcfasm would be coming from pip, and it is as a "+ third_party/..." package.